pkg/sdk/resourcebuilder/component.go

```diff
@@ -31,14 +31,17 @@ import (
 	crdv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	serializer "k8s.io/apimachinery/pkg/runtime/serializer/json"
+	k8stypes "k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/intstr"
 	"sigs.k8s.io/controller-runtime/pkg/builder"
 )
 
 const (
-	Image            = "ghcr.io/banzaicloud/logging-operator:3.10.0"
+	Image            = "ghcr.io/banzaicloud/logging-operator:3.9.5-dev"
```

Comment:
Is this the image version we're going with?

Reply:
No, I think we will need an even newer tagged image for this, possibly 3.11.0